app.js

```diff
@@ -124,10 +124,16 @@ Test this function by hand in the console to get it working, and when you think
 
 // Write your code here
 function multiplyArray(testArray){ //eslint-disable-line
+  var calculatedMultiply = multiply(testArray[0], parseInt(multiply(testArray[1],testArray[2])[0]))[0];
```

Comment:
This does make the test pass, but what if you use a `testArray` with more than three elements?